pkg/services/rendering/rendering.go

```diff
@@ -440,6 +440,15 @@ func (rs *RenderingService) getGrafanaCallbackURL(path string) string {
 		subPath = rs.Cfg.AppSubURL
 	}
 
+	if rs.Cfg.RendererCallbackUrl != "" {
+		u, err := url.Parse(rs.Cfg.RendererCallbackUrl)
+		if err != nil {
+			// Error handling here if renderer callback url is not a valid URL?
```

Comment:
We could add a warning log here, but we allow Grafana to start even when the Image renderer is not configured.